### PR TITLE
Add test for gl_VertexID.

### PR DESCRIPTION
**Pull request by jdashg**
In response to #2770, tests drawArrays, drawElements, and drawArrays+transform-feedback.

### CODE REVIEW

**Comment by jdashg** [comment]
FWIW, this fails in Firefox+Windows+ANGLE, but mostly works for Firefox+Windows+GL. The drawElements tests fail even with GL, but I think that's a Firefox bug.

File: sdk/tests/conformance2/rendering/vertex-id.html [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+</script>
+<script src="../../js/js-test-post.js"></script>
+
+</body>
+</html>
+
+<!--
+
+/*
+** Copyright (c) 2013 The Khronos Group Inc.
```
Comment (kenrussell) [inline sdk/tests/conformance2/rendering/vertex-id.html]: Please move closer to the top of the file and update the copyright date.

Comment (jdashg) [reply sdk/tests/conformance2/rendering/vertex-id.html]: Let's talk briefly about this on the concall tomorrow.

**Comment by jdashg** [comment]
Decision from the concall was to tentatively allow this, but try to move towards a smaller copyright block that does not incur charset reparse so easily. (Point people at a COPYRIGHT.txt?)

**Pull request merged**